<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="preControlTab">
       <attribute name="title">
        <string>Pre Control</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>740</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="buttomsLayout" rowstretch="0,0" columnstretch="0,4,0,4,0,4">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="1">
          <widget class="QComboBox" name="groupCB"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="familyL">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Family:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="solderPasteCB">
           <item>
            <property name="text">
             <string>Select Paste...</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Indium 5.8LS Lead free</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Indium 8.9E Lead free</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Senju M40-LS720V-HF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Alpha OM 338 PT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Indium SMQ92H (Sn63Pb37)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="solderPasteL">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Solder Paste:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QComboBox" name="boardSideCB">
           <item>
            <property name="text">
             <string>Top</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bottom</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="groupL">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Group:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="modelCB"/>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="selectFilePB">
           <property name="text">
            <string>Select PDF</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="pdfFileL">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;PDF File:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="modelL">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Model:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="familyCB">
           <item>
            <property name="text">
             <string>Select Family...</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Video</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DSR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pics</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>E6mil</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="boardSideL">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Board Side:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTableWidget" name="parametersTW">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>361</width>
          <height>211</height>
         </rect>
        </property>
        <row>
         <property name="text">
          <string>Minimum Time Above Liquidus (sec)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Maximum Time Above Liquidus (sec)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Minimum peak temperature (째C)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Maximum peak temperature (째C)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Minimum rise slope (째C/sec)</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Maximum rise slope (째C/sec)</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string notr="true">ACCEPTABLE PARAMETERS</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </column>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>170</y>
          <width>281</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="resultsLayout">
         <item>
          <widget class="QPushButton" name="positiveSlopePB">
           <property name="text">
            <string>Positive Slope Results</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="TALPB">
           <property name="text">
            <string>TAL Results</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="peakTempPB">
           <property name="text">
            <string>Peak Temperature Results</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
